lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/data/ashrae_90_1_2007.spc_typ.json

```diff
@@ -7244,14 +6548,6 @@
       "service_water_heating_fraction_sensible": 0.2,
       "service_water_heating_fraction_latent": 0.05,
       "service_water_heating_schedule": "HotelLarge LaundryRoom_SWH_Sch_Post2004",
-      "exhaust_per_area": null,
-      "exhaust_fan_efficiency": 0.16,
-      "exhaust_fan_power": null,
-      "exhaust_fan_pressure_rise": 125.0,
-      "exhaust_fan_maximum_flow_rate": 500.0,
-      "exhaust_availability_schedule": "HotelLarge Hours_of_operation",
-      "exhaust_flow_fraction_schedule": "HotelLarge Hours_of_operation",
-      "balanced_exhaust_fraction_schedule": null,
```

Comment:
This entry is for 

```json
"template": "90.1-2007",
"building_type": "LargeHotel",
"space_type": "Laundry",
```

it has non null values, BUT exhaust_per_area is null. Which gives it no exhaust fan, because that was the main test key

In fact, Only the DOE Ref PRe 1980 template has a laundry exhaust fan applied in previous openstudio-standards version (I tested this with openstudio-standards 0.5.0).


so if I'm following this PR correctly:
* Any ASHRAE 90.1 template LargeHotel - Laundry used to have zero exhaust fan beforehand
* Now it would have a 0.7 CFM/ft² one

So my questions @eringold and @mdahlhausen, is that:

* Am I following correctly?
* If so, is this is intended (meaning the previous implementation was faulty) or not?

thanks!